scripts/nm-quick.sh

```diff
@@ -582,7 +653,21 @@ install_netmaker() {
 
 	# start docker and rebuild containers / networks
 	cd "${SCRIPT_DIR}"
-	docker-compose up -d --force-recreate
+	if [ -f /etc/debian_version ]; then
+		docker compose up -d --force-recreate
+	elif [ -f /etc/fedora-release ]; then
+		docker compose up -d --force-recreate
+	elif [ -f /etc/amazon-linux-release ]; then
+		docker compose up -d --force-recreate
+	elif [ -f /etc/centos-release ]; then
+		docker compose up -d --force-recreate
+	elif [ -f /etc/redhat-release ]; then
+		docker compose up -d --force-recreate
+	elif [ "$(cat /etc/*-release |grep suse |wc -l)" -gt 0 ]; then
+		docker compose up -d --force-recreate
+	else
+		docker-compose up -d --force-recreate
```

Comment:
why do we still need `docker-compose`?

Reply:
In the old code logic, there are the code for Alpine, ArchLinux and BSD,  they are not covered in the test. Because there is no cloud version for these OSs.  There is no public ip for local vm, the installation process could not complete.  The "docker-compose" is for these unchanged OSs.

The changes and test covered:
Ubuntu
Debian
Fedora
Centos
Redhat
Amazon Linux
Rocky Linux
AlmaLinux

SUSE Linux is tested, but it does not work. There is no x86_64 docker packages for SUSE, only os390 version supported.
@abhishek9686

Reply:
Good point.  Deleted suse and added the OS support list if the OS is not in the list.